Doc/reference/expressions.rst

```diff
@@ -1308,8 +1326,12 @@ The shifting operations have lower priority than the arithmetic operations:
 .. productionlist:: python-grammar
    shift_expr: `a_expr` | `shift_expr` ("<<" | ">>") `a_expr`
 
-These operators accept integers as arguments.  They shift the first argument to
-the left or right by the number of bits given by the second argument.
+These operators accept integers as arguments.  In the former case they shift the
+first argument to the left or right by the number of bits given by the second
+argument.
```

Comment:
Leaving this diff made it easy to miss that your text still says "In the former case" here.